src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "N+1 API Calls"
+sidebar_order: 15
+description: "Learn more about N+1 API calls and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+_N+1 API Calls_ are a performance problem in which an application makes many simultaneous network requests to fetch a given type of resource. For example, an application might make one network request for each item in a list, instead of fetching all the information in a combined network call. Each call to the server incurs some performance overhead, and if the number of calls is high this can lead to severe performance problems. This problems commonly occurs when rendering lists or grids of items, where each item needs to fetch additional information from the server.
+
+## Detection Criteria
+
+The N+1 API Calls detector looks for groups of simultaneous network calls to the same URL. It uses has the following criteria for detection:
+
+- applies to `GET` requests only
+- a minimum of 10 network calls need to happen at the same time
+- the network calls must be a minimum of 50s in duration, and within 5ms of each other
+- the calls must not be to a GraphQL endpoint, or to `_next/data`
+- the calls must not be fetching a static resource (e.g., `.png` files)
+
+## Span Evidence
+
+The evidence for an N+1 queries problem has three main aspects:
+
+- Transaction name
+- Repeating spans - This is the URL that's receiving the repeated network calls
+- Parameters - These are the query parameters that vary between the network calls
+
+![N+1 API Calls span evidence](n-plus-one-api-calls-evidence.png)
```

Comment:
```suggestion
![N+1 API Calls span evidence](n-plus-one-api-calls-evidence.png)

View it by going to the **Issues** page in Sentry, selecting your Android project, clicking on the N+1 API Call error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.


```
I recommend adding this section to all 3 performance issue types (N+1 Queries, N+1 API calls, and File IO). I think it's helpful for the user to know where exactly to go to see this in the product. I'm assuming these errors will be called N+1 API Calls?

Reply:
They will be called that yeah, but I want to ask about this suggestion! People will land on this page from either:

1. Clicking the "Read More About N+1 API Call Issues" _from_ the issue details page
2. Going through the docs from Issues → Issue Details → Performance Issues → N+1 API Calls
3. Googling?

I think it might be weird to make a point of explaining how to get to a page that you probably came from?

Reply:
Super valid point. One of my goals is to get better analytics set up that trace how people go from the product to docs and back. That would answer these kinds of questions. Absolutely agree that it doesn't make sense for 1. & 2. But I think it does make sense for 3. To me, people who are coming from the product will just ignore the instructions for where to find it, but it may be SUPER useful to people who are coming from Google and may not know where to go. Plus, I think the product UI is not super intuitive and it can be hard to find things unless you've done it over and over again. So to me, the reward of helping people find the thing we're talking about in the product, outweighs the risk of being redundant for those who are coming from there. What do you think?

Reply:
That makes sense! The only strange aspect, I think, is that we're telling people to go to a page that probably doesn't exist. I'm worried that the doc has specific instructions that probably go nowhere, since most people won't have this kind of issue, or will need to search for it, or might not even be a Sentry customer! This isn't really a _feature_ that is always present, it's an explainer for a type of issue they might have 🤔

Would something like this work?

> You can find the "Span Evidence" section on the issue details page of an N+1 API Call issue, in the "Details" tab.

Or is that too sparse?

Reply:
I added your suggestion minus the "Android" part, since it's not relevant here 🙏🏻